sha3/src/main/java/com/theromus/exception/CantSetCustomHashLen.java

```diff
@@ -0,0 +1,9 @@
+package com.theromus.exception;
+
+@SuppressWarnings("serial")
+public class CantSetCustomHashLen extends Exception{
```

Comment:
Remove this class. Let's use one Exception - NotValidHashLenException

Reply:
This was to have a more readable error, but I can add a message in the Exception - NotValidHashLenException.